security/tinc/src/opnsense/scripts/OPNsense/Tinc/lib/objects.py

```diff
@@ -137,7 +139,10 @@ def set_connectto(self, value):
 
     def config_text(self):
         result = list()
-        result.append('Address=%(address)s %(port)s'%self._payload)
+        if 'address' in self._payload:
+            addresses = self._payload['address'].split(',')
+            for address in addresses:
```

Comment:
same

Reply:
Was micmic of existing code for subnets...
Reworked both this way.

Reply:
@ElNounch nice!